src/command_modules/azure-cli-security/azure/cli/command_modules/security/custom.py

```diff
@@ -0,0 +1,455 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.mgmt.security.models import SecurityContact, AutoProvision
+
+# --------------------------------------------------------------------------------------------
+# Security Tasks
+# --------------------------------------------------------------------------------------------
+
+
+def list_security_tasks(
+        client,
+        resource_group_name=None):
+
+    for loc in client.locations.list():
+        client.config.asc_location = loc.name
```

Comment:
Why are you setting the location to the last one in the list?

Reply:
there is always one location in location. this is our "home region" and this is where we store some of the data of the user.